### ISSUE TITLE
Add alt for svg icons

### ISSUE THREAD

**Issue by zwiastunsw** [darekkay/evaluatory#5]
Icons that indicate the severity of a problem (critical, serious, etc.) should have an alt attribute. 
The report should be correct in terms of accessibility

**Comment by darekkay** [comment]
Thanks for this finding! I don't know how this one slipped through.

**Comment by zwiastunsw** [comment]
Yes, it should be the title element or the text element, not the alt attribute :)
There is a title element in all icon files. But in the result code of the results page this element is no longer there. Why? I don't know and I don't understand. It is possible that the problem is the generated sprite.svg file.

**Comment by darekkay** [comment]
Fixed and released in `2.3.1`.
